votey/models.py

```diff
@@ -35,11 +35,13 @@ class Option(db.Model):  # type: ignore
     id: int = db.Column(db.Integer, primary_key=True)
     poll_id: int = db.Column(db.Integer, db.ForeignKey("poll.id"), nullable=False)
     option_text: str = db.Column(db.Text, nullable=False)
+    option_emoji: str = db.Column(db.Text, nullable=False)
```

Comment:
i think it makes more sense to store this as a nullable string and treat it as an `Optional[str]` in python land. we don't know, but imagine a world where users can update or remove or reorder poll options: if we persist the numbered emoji to the DB, we now have to go through the DB and fix up all of the emojis. if we use a nullable string, then `null/None` indicates to just use the index in the array *at the time of use* rather than at the time of poll definition.